apps/dashboard/app/helpers/dashboard_helper.rb

```diff
@@ -10,7 +10,14 @@ def logo_image_tag(url)
     if url
       uri = Addressable::URI.parse(url)
       uri.query_values = (uri.query_values || {}).merge({timestamp: Time.now.to_i})
-      %(<img src="#{uri}" alt="logo" />).html_safe
+      logo_css_attributes = "margin-bottom: 10px;"
+
+      if uri.extname == ".svg"
```

Comment:
Since there is no default svg value we could just drop the conditional. Then you can just control the height of the logo if you really want to, regardless of the image.

Also, you could make logo_height's default value be "auto". Then a site setting the value via env var would need to do 85px not 85 but then you can just do: `height: #{Configuration.logo_height}` without the conditional.

Finally, the tag helper https://apidock.com/rails/v5.2.3/ActionView/Helpers/TagHelper/tag might be used in place of the string interpolation for the image tag